tests/multigrid/transfer_matrix_free_06.cc

```diff
@@ -43,7 +43,7 @@ check(const unsigned int fe_degree)
   deallog << "FE: " << fe.get_name() << std::endl;
 
   // run a few different sizes...
-  unsigned int sizes[] = {1, 3};
+  unsigned int sizes[] = {1, 2};
```

Comment:
Could you please also change line 59 to `if (degree == 1)`? We have some higher degrees already, I do not think we need more mesh elements.